src/components/organisms/NeynarCastCard/index.tsx

```diff
@@ -32,20 +30,22 @@ export type NeynarCastCardProps = {
   identifier: string;
   viewerFid?: number;
   allowReactions: boolean;
```

Comment:
Can we have a default value for `allowReactions`, maybe set it to true?
This will reduce the number of required props, one less step for devs to get it running. Will improve devEx

Also can we align the share button and `replies and likes` when allowReactions is set to `false`
<img width="688" alt="image" src="https://github.com/user-attachments/assets/968a80a5-d984-4ced-a3d5-7a67d15f2795">